pkg/util/log/crash_reporting.go

```diff
@@ -37,6 +37,12 @@ import (
 )
 
 var (
+	// versionTag is expected to be set by our build process for the binaries
+	// which we actually expect to report crashes (i.e. official channels). It's
+	// the same as the build.Tag for published releases, and a simplified version
+	// otherwise that should result in those errors aggregating better.
+	versionTag = "development"
```

Comment:
If you take the above suggestion, let's call this "unknown-dev" too so that unknown builds all end up in the same env.